src/test/java/org/apache/accumulo/proxy/its/ProxyDurabilityIT.java

```diff
@@ -98,6 +98,10 @@ public void testDurability() throws Exception {
       proxyProps.put("tokenClass", PasswordToken.class.getName());
       proxyProps.putAll(getClientProperties());
 
+      String login = ROOT_PASSWORD;
+
+      proxyProps.put("secret", login);
+
```

Comment:
"secret" and "login" are significantly overused as variables, constants, etc. It's very hard to follow the tests, when the root password is being stored in a variable called "login" that is actually the shared secret for authenticating to the proxy. There needs to be greater clarity between all of these, with better variable names, and better test values, and less reuse between them... I suspect some tests "just work" accidentally, because we used "secret" more than once, so even though we're probably passing the wrong variable, it still passes, because both values are "secret". Everything should have unique values.